### DISCUSSION TITLE
2023-09-04

### DISCUSSION THREAD

**Discussion by jaehee329** [woowacourse-teams/2023-haru-study#473] [daily_meeting]
### 오늘의 할 일
개인 미션 진행(백 제출 월,  프 제출 목)
### 내일 할 일
1. 사용자 유치 계획 (9/8)
2. 기존 개인용 서비스를 유지하는 방향에 대해 논의

**Comment by nlom0218** [comment]
### 체크인

9점 - 날씨가 점점 풀려서 좋아요

**Comment by yeopto** [comment]
### 체크인

10점 - 다시 태어난 기분입니다.

### 체크아웃

10점 - 바쁘게 하루가 간거 같아서 기분이 좋네요

**Reply by woosung1223** [reply]
응애~ 나 아가엽토

**Comment by aak2075** [comment]
### 체크인

10점 - 정신이 혼미해서 좋아요

### 체크아웃

10점 - 그래도 오늘 하루 잘 마무리 했네요

**Comment by jaehee329** [comment]
### 체크인

8점 - 주말에 못 쉬었어요.. ㅠ

### 체크아웃
7점 - 또 하루 보냈다!

**Comment by woo-jk** [comment]
### 체크인 
8점 - 적당히 피곤해요

### 체크아웃
8점 - 할게 너무 많아요

**Comment by woosung1223** [comment]
### 체크인
8점 - 월요일이지만 커피를 마셨지만 피곤하지만 그냥 그럭저럭 괜찮습니다

### 체크아웃
9점 - 오늘 어느정도 할 일은 마친 것 같아 기분이 좋습니다!

**Comment by MoonJeWoong** [comment]
### 체크인
7점 - 미션이 아직 남아서 -2점, 할게 참 많아서 -1점입니다. 얼른 마무리하고 싶네용

### 체크아웃
8점 - 미션 완료해서 1점 올랐으나 만족할만큼 리팩토링하진 못한 것 같아 아쉽네용
